exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -159,7 +170,13 @@ export async function handler (
       : undefined
     return printProjectCommands(manifest, rootManifest ?? undefined)
   }
-  if (scriptName !== 'start' && !manifest.scripts?.[scriptName]) {
+
+  const multiScriptSelectorSpecified = scriptName.slice(-2) === ':*'
```

Comment:
I don't understand this. You switched to a regex but still use `:*`? How so?

Reply:
For example, a regular expression `build` without wildcard selector will match `build:css` and this is against intention. Therefore, I left this boolean to determine if it is wildcard mode or not.

However I rethought about it and this behavior is against intention as above description and if we want to use RegExp for script matching, a detailed condition judgement process is needed and I thought it is clearer and more maintainable to match script name with simple string prefix matching ( `startsWith` ) instaed of matching with RegExp so I revert the commit to switch to regex in [7608448](https://github.com/pnpm/pnpm/pull/5871/commits/7608448ec57c581254a373e4ef79ad7146303586).

Reply:
After all,  `multiScriptSelectorSpecified` is still needed whether  RegExp is used or not. (In the current implementation, RegExp is not used after all.)